src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/IPackageMetadata.cs

```diff
@@ -48,5 +49,7 @@ public interface IPackageMetadata
         /// Returns sets of Content Files specified in the manifest.
         /// </summary>
         ICollection<ManifestContentFiles> ContentFiles { get; }
+
+        IReadOnlyList<PackageType> PackageTypes { get; }
```

Comment:
Cool, choose whatever you think looks best

---
In reply to: [63775443](https://github.com/NuGet/NuGet.Client/pull/603#discussion_r63775443) [](ancestors = 63775443)